LICENSE

```diff
@@ -0,0 +1,22 @@
+(The MIT License)
+
+Copyright (c) 2015 Apiary LTD. <support@apiary.io>.
```

Comment:
@kylef after a brief chat with Ondrej it seems we should be using Inc. Unless @Almad thinks otherwise. Please change it to inc.